### PR TITLE
[OSDOCS-11242]: HCP updates on Welcome page

### PR DESCRIPTION
**Pull request by lahinson**
**This PR is for the RHACM 2.11 release that is scheduled for July 18. Do not merge until this date.**

Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-11242

Link to docs preview: https://78892--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/index.html#hosted-control-plane-activities (links to the RHACM docs will not work until July 18)

QE review:
- [x] QE has approved this change.

Additional information: Change management has been completed for this PR.

### CODE REVIEW

**Comment by xltian** [comment]
LGTM

**Comment by zanetworker** [comment]
LGTM

File: welcome/index.adoc [hunk 1]
```diff
@@ -325,15 +325,13 @@ endif::openshift-rosa[]
 ifdef::openshift-enterprise[]
 == Hosted control plane activities
 
-* **Support for bare metal and {VirtProductName}**: {hcp-capital} for {product-title} is now Generally Available on bare metal and {VirtProductName} platforms. For more information, see the following documentation:
+* **Support for {aws-first}**: {hcp-capital} for {product-title} is now Generally Available on the {aws-short} platform. For more information, see the following documentation:
 
-** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-bm[Configuring hosted control plane clusters on bare metal]
-** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#hosted-control-planes-manage-kubevirt[Managing hosted control plane clusters on OpenShift Virtualization]
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosting cluster on {aws-short}]
```
Comment (jneczypor) [inline welcome/index.adoc:330]: I get a 404 error for this link.

File: welcome/index.adoc [hunk 2]
```diff
@@ -325,15 +325,13 @@ endif::openshift-rosa[]
...
-** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosted cluster on 64-bit x86 bare metal for {ibm-z-name} compute nodes (Technology Preview)]
-** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-agent-non-bm[Configuring hosted control plane clusters using non bare metal agent machines (Technology Preview)]
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#config-hosted-service-ibmpower[Configuring the hosting cluster on a 64-bit x86 {product-title} cluster to create {hcp} for {ibm-power-name} compute nodes (Technology Preview)]
```
Comment (jneczypor) [inline welcome/index.adoc:334]: I get a 404 error for this link.

File: welcome/index.adoc [hunk 3]
```diff
@@ -325,15 +325,13 @@ endif::openshift-rosa[]
...
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosted cluster on 64-bit x86 bare metal for {ibm-z-name} compute nodes (Technology Preview)]
```
Comment (jneczypor) [inline welcome/index.adoc:335]: I get a 404 error for this link.

File: welcome/index.adoc [hunk 4]
```diff
@@ -325,15 +325,13 @@ endif::openshift-rosa[]
...
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-configure-agent-non-bm[Configuring hosted control plane clusters using non bare metal agent machines (Technology Preview)]
```
Comment (jneczypor) [inline welcome/index.adoc:336]: I get a 404 error for this link.

**Review by jneczypor — COMMENTED** [review]
I get 404 errors for all the new links. Besides that and one tiny change, it looks good!
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: welcome/index.adoc (hunk 1 above)
Comment (lahinson) [reply welcome/index.adoc:330]: Yep. As I mentioned in the first comment of this PR, none of the links will work until July 18. Thanks!

**Pull request merged**